controls/radvirtualgrid/mvvm.md

```diff
@@ -0,0 +1,327 @@
+---
```

Comment:
Can we move this into a new folder called Populating with Data or something like this?